src/Components/App/App.js

```diff
@@ -1,20 +1,52 @@
 import './App.scss';
-import {Homepage} from '../HomePage/Homepage'
-import Header from '../Header/Header'
-import React, {Component} from 'react'
-import {Route} from 'react-router-dom'
+import {Homepage} from '../HomePage/Homepage';
+import Header from '../Header/Header';
+import { Form } from '../Form/Form';
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
 
 class App extends Component {
   constructor() {
-    super()
+    super();
+    this.state = {
+      activities: [],
+      musicAnswers:[],
+      moviesAnswers:[],
+      podcastsAnswers:[],
+      boardGamesAnswers:[],
+      cardGamesAnswers:[]
+    }
   }
+
+  findSetOfQuestions = () => {
```

Comment:
I thought we got rid of `findSetOfQuestions` ?

Reply:
That is weird, I think I did. I might have nos committed the changes by mistake, I will take care of it.

Reply:
No worries!